libs/heattracesidesheet/src/lib/utils-sidesheet/useGetHeatedTags.ts

```diff
@@ -0,0 +1,29 @@
+import { useHttpClient } from '@equinor/fusion-framework-react-app/http';
+import { useContextId } from '@cc-components/shared';
+import { useQuery } from '@tanstack/react-query';
+import { HeatTraceHeatedTag } from '@cc-components/heattraceshared';
+
+export const useGetHeatedTags = (heatTraceCabelId: string) => {
+  const client = useHttpClient('cc-api');
```

Comment:
Prefer to use the useHttpClient from the shared so you dont have to specify "cc-api" this makes it easier in the future

Reply:
Fixed.